discussment-ui-wicket/src/main/java/org/danekja/discussment/ui/wicket/list/post/PostListPanel.java

```diff
@@ -101,7 +103,7 @@ protected String load() {
                     }
                 }));
 
-                listItem.add(new Label("created", new PropertyModel<String>(listItem.getModel(), "created")));
+                listItem.add(DateLabel.forDateStyle("created", new PropertyModel<Date>(listItem.getModel(), "created"), "MS"));
```

Comment:
What does MS stand for?

Reply:
The first letter is for format of date, the second for time where M stands for medium, S for short. The exact format depends on locale set.

Details at https://www.joda.org/joda-time/apidocs/org/joda/time/format/DateTimeFormat.html#forStyle-java.lang.String-